guides/common/modules/con_comparing-multiple-content-view-environments-and-composite-content-views.adoc

```diff
@@ -0,0 +1,20 @@
+[id="comparing_multiple_content_view_environments_and_composite_content_views_{context}"]
+= Comparing multiple content view environments and composite content views
```

Comment:
Composite CVs are covered in the Content guide, so this seems another good reason to move these modules to that guide.